default/stringtables/en.txt

```diff
@@ -10687,6 +10687,52 @@ Invincible'''
 
 ### End of Capital Names Lists
 
+### AI Diplomacy Strings and Lists
+### Translator Note: for AI Keys ending with "LIST", each line of the provided value should be an independent entry; each entry must be a single line.
+### For such lists, you may provide more or fewer entries than are provided in en.txt, unless otherwise indicated in a comment for that key
+
+AI_PEACE_PROPOSAL_ACKNOWLEDGEMENTS_MILD_LIST
+'''Nice to hear from you again.
+Receipt of transmission acknowledged.
+'''
+
+AI_PEACE_PROPOSAL_ACKNOWLEDGEMENTS_HARSH_LIST
+'''What? Aren't you dead yet?
+Oh, not another pathetic sob story, please!
+'''
+
+AI_PEACE_PROPOSAL_RESPONSES_YES_MILD_LIST
+'''We will be happy to trust your good intentions.
+Whatever you say, Boss.  Everyone says you are very trustworthy.
+'''
+
+AI_PEACE_PROPOSAL_RESPONSES_NO_MILD_LIST
+'''Unfortunately, that would be very inconvenient at this time.
+We regret to inform you that we are unable to comply with that request.
+'''
+
+AI_PEACE_PROPOSAL_RESPONSES_YES_HARSH_LIST
+'''Out of pity, we will agree to not attack you.  For now.
+Feeble Beings!  There would be little honor in vanquishing you.
+'''
+
+AI_PEACE_PROPOSAL_RESPONSES_NO_HARSH_LIST
+'''We will see you dead first!
+You will not have peace until all your base r belong to us!
```

Comment:
No, it's a slang-ish gaming expression, commonly used as a challenging assertion, "All your base r belong to us!"  (It may have originally been partly a mistype, though.)